test/validators.js

```diff
@@ -6230,6 +6228,8 @@ describe('Validators', () => {
           '17012345678',
           '+4912345678910',
           '+49015123456789',
+          '015345678910',
```

Comment:
https://de.wikipedia.org/wiki/Vorwahl_01_(Deutschland)#015 153 and 154 are not valid numbers.